docs/kjar/README.md

```diff
@@ -0,0 +1,75 @@
+### Kjar
+
+#### Deploy kjar at startup from the fat jar
+
+This is the default configuration using the sample-hacep-project/sample-hacep-project-kjar maven module, 
+no env vars are required but it can't be updated at runtime.
+
+#### Deploy kjar at startup from Maven Repo
+
+To enable the update at startup and at runtime some ENV VArs are needed in the deployment.yaml
+and the dependency 
+```xml
+<dependency>
+      <groupId>org.kie</groupId>
+      <artifactId>sample-hacep-project-kjar</artifactId>
+</dependency>
+```
+must be removed from you maven module,
```

Comment:
Typo "you" -> "your". Please write from which module exactly.

Reply:
Ok